ui/window.go

```diff
@@ -157,6 +158,10 @@ func (win *Window) Loop(shutdown chan int) {
 					win.updateSyncStatus(true, false)
 				}
 			case wallet.SyncCanceled:
+				if win.sysDestroyWithSync {
+					close(shutdown)
+					return
+				}
```

Comment:
This seems redundant. The shutdown channel seems to already be closed when the system DestroyEvent is triggered.